catkin_tools/context.py

```diff
@@ -411,20 +446,20 @@ def summary(self, notes=[]):
                 "--init`.")]
         if not self.source_space_exists():
             summary_warnings += [clr(
-                "Source space `@{yf}{_Context__source_space_abs}@|` does not yet exist.")]
+                "Source space `@{yf}{__source_space_abs}@|` does not yet exist.")]
```

Comment:
I would have though so, but this was what worked. It may be an indication that I'm not using `property` correctly in the setup of these generated properties. I shall investigate and report back.